.golangci.example.yml

```diff
@@ -213,7 +213,9 @@ linters-settings:
   dogsled:
     # checks assignments with too many blank identifiers; default is 2
     max-blank-identifiers: 2
-
+  testpackage:
+    # regexp pattern to skip files
+    skip-regexp: (export|internal)_test\.go
```

Comment:
Can we do same with `exlude-rules`? Seems like this option is excessive, what do you think?

Reply:
`(export|internal)_test\.go` is default skip pattern for testpackage linter. And this default is good enough for most projects. You should change it in very rare cases.

Correct me if I'm wrong. If we replace it with `exclude-rules`, we have to define default pattern for each new project explicitly.

That's why I've added this setting.

Reply:
Oh, now I see, thank you.